src/app/screens/LNURLPay/index.tsx

```diff
@@ -341,92 +341,98 @@ function LNURLPay() {
                   description={getRecipient()}
                   image={navState.origin?.icon || getImage()}
                 />
-
-                <div className="my-4">
-                  <dl>
-                    <>
-                      {formattedMetadata(details.metadata).map(
-                        ([dt, dd], i) => (
-                          <Fragment key={`element-${i}`}>
-                            <Dt>{dt}</Dt>
-                            <Dd>{dd}</Dd>
-                          </Fragment>
-                        )
-                      )}
-                      {details.minSendable === details.maxSendable && (
+                <form>
```

Comment:
Can the confirm-action be moved to the form-submit like we do it here or in most form-situations?:  https://github.com/vaibhavgarg237/lightning-browser-extension/blob/disableButtons/src/app/screens/Unlock/index.tsx#L63
Would be nice to align this.

Reply:
Actually  because of [this](https://github.com/getAlby/lightning-browser-extension/blob/a7c2332aeb8bb887e1c8b54af5611580d0c3cc52/src/app/components/ConfirmOrCancel/index.tsx#L35), it works same as expected, like on pressing enter it triggers Confirm-action( user can send the form using enter ).

`
        <Button
          type="submit"
          onClick={onConfirm}
`

But for aligning it like rest of code base we can move it to handleSubmit like implementation. Will do that...

Reply:
Yeah, you're right, but aligning it makes it more clear and put's the action where it should be I believe.

Reply:
Okay, will do that.

Reply:
@escapedcat I have updated the changes.